app/views/users/show.html.erb

```diff
@@ -1 +1,12 @@
-<h1>Show specific user</h1>
\ No newline at end of file
+<section class="align">
+	<%= render 'shared/user', user: @user %>
+	<div class="align">
+		<p><%= @user.bio %></p>
+		<% @user.most_recent_posts.each do |post| %>
+		 <%= render 'shared/post', post:post %>
+		<%end %>
+	</div>
+	<div>
+		<button><%= link_to "see all posts", user_posts_path(@user) %></button>
+	</div>
+</section>
```

Comment:
- [ ] [REQUIRED] Great job on the User show page so far @Ridwanullahi-code  👍🏽 However, we need to put in more effort to look like the wireframe provided.
### Your Design
<img width="1440" alt="Screenshot 2023-04-15 at 19 09 00" src="https://user-images.githubusercontent.com/58692590/232246773-8f92ff01-49c3-4eb5-9ef2-310a7940b1d8.png">

### Design Template
<img width="474" alt="Screenshot 2023-04-15 at 19 16 00" src="https://user-images.githubusercontent.com/58692590/232246708-4c4126af-e55a-4946-8f77-f38bd3e8f99c.png">